### ISSUE TITLE
argparse.ArgumentParser must have description

### ISSUE THREAD

**Issue by orsinium** [wemake-services/wemake-python-styleguide#1034]
# Rule request

## Thesis

```python
# bad
parser = ArgumentParser()
# good
parser = ArgumentParser(description="do magic")
```

## Reasoning

flake8-docstrings enforces docstrings for everything that part of your public interface. CLI is definitely the most important part of the API, and it should have a good description as well.

**Comment by sobolevn** [comment]
Well, I guess it is too-specific. And it would be hard to check that we cover all possible `ArgumentParser` use-cases. Maybe some `flake8-argparse` would be a better place for this rule?
